### PR TITLE
Copy & Test Updates

### PR DESCRIPTION
**Pull request by xavdid**
Fixes that happened in other sample apps, but not here (since it hadn't been added at the time). Also adds a light readme and moves a test value into the code itself. Matching changes in https://github.com/zapier/zapier/pull/16752.

### CODE REVIEW

File: test/issue.js [hunk 1]
```diff
@@ -6,6 +6,8 @@ const zapier = require('zapier-platform-core');
 const App = require('../index');
 const appTester = zapier.createAppTester(App);
 
+const REPO = 'username/reponame' // CHANGE THIS
```
Comment (eliangcs) [inline test/issue.js]: Devs will need to change code to make the test pass. Wouldn't it be better if we keep it as an environment variable in `.env`?

Comment (xavdid) [reply test/issue.js]: That was the value it was replacing, so I left it basically as is. 

I added copy in the `z/z` PR in the description to update this file, but I could also see setting the initial value to this repo so the tests run as written. 

Alternatively, we could make a test repo that we ignore, then we don't care if they open issues. 

What do you think?

Comment (xavdid) [reply test/issue.js]: The main reason to move it was that it's not a secret value (which is typically what env is for). It's a per-test configuration, so it makes sense to set it that way.

Comment (eliangcs) [reply test/issue.js]: I personally prefer the original setup: having a `TEST_REPO` in the `.env` file, allowing the devs to set it to their personal repo without changing the code. @BrunoBernardino what do you think?

Comment (BrunoBernardino) [reply test/issue.js]: I personally prefer the `.env` approach as well, but I guess @xavdid has a valid point with it not being secret and usually set once, so it's your call.

**Comment by xavdid** [comment]
Awesome, appreciate the review! 

To me, contents of the `.env` file should be global (`client_id`) and/or secret (`client_secret`, `api_key`). 

It would feel weird to have other test configurations (such as issue title or body) anywhere besides the test itself, so it makes sense to me to include repo in that as well. Plus, getting the user to look in and tweak an easy thing in the test file seems like a win for code familiarity and learning.

Given all that, I'm going to leave as written. If we need to change it, we can.

**Pull request merged**